cfme/tests/infrastructure/test_infra_manual.py

```diff
@@ -288,39 +288,234 @@ def test_host_comparison_my_company_tags():
+                5. Click "Download as CSV" under the Downloads pulldown.
+                6. Click "Print or export as PDF".
+            expectedResults:
+                1. Hosts view is displayed.
+                2. The selected hosts should be displayed with a blue border and checked checkbox.
+                3. The "Compare Host / Node" view should be displayed.
+                4. A compare report should be downloaded (Chrome will show it in left bottom
+                   corner of window.
+                   - report is in text format.
+                   - report has date appended to name.
+                   - data in the report matches the GUI.
+                5. A compare report should be downloaded (Chrome will show it in left bottom
+                   corner of window.
+                   - report is in CSV format.
+                   - report has date appended to name.
+                   - data in the report matches the GUI.
+                6. A print page is displayed containing the compare report.
+    """
+    pass
+
+
+@test_requirements.infra_hosts
+def test_host_comparison_multipleviews_interactions():
+    """
+    Here i will test adding and removing rows and sections in permutations I did not cover
+    explictily already. Also cover the case where clicking on the parent section selects all of
+    the child sections.
+
+        Polarion:
+                assignee: prichard
+                casecomponent: Infra
+                caseimportance: low
+                initialEstimate: 1/4h
+                testSteps:
+                    1. Navigate to the Compute > Infrastructure > Hosts view.
+                    2. Select at least 3 hosts (more if possible) by checking the box in upper left
+                       of quadicons.
+                    3. Click "Compare Selected Items" under the "Configuration" dropdown.
+                    4. Click to select "My Company Tags" in the Comparison Sections dropdown,
+                       and click "Apply".
+                    5. Repeat step 4 for Configuration, Security, and Properties sections.
+                       Note: Properties will need to be clicked twice as it defaults to checked
+                       with only one section enbled.
+                    6. Click on "My Company Tags" in Comparison Sections dropdown.
+                    7. Click on a child item, to deselect, and click "Apply".
+                    8. Click on random child items within the section and click "Apply" until no
+                       child items remain selected.
+                    9. Click on random child items within the section and click "Apply" until all
+                       child items remain selected.
+                    10. Repeat steps 6 through 9 for Configuration, Security, and Properties
+                        sections.
+
+                expectedResults:icon
+                    1. Hosts view is displayed.
```

Comment:
Typo: ```expectedResults:icon```
I think, you can remove that extra ```icon```. Also you can remove block line(504)

Reply:
Updated as suggested.